cpp/src/arrow/io/test-common.cc

```diff
@@ -115,8 +115,9 @@ void MemoryMapFixture::CreateFile(const std::string& path, int64_t size) {
 }
 
 Status MemoryMapFixture::InitMemoryMap(int64_t size, const std::string& path,
-                                       std::shared_ptr<MemoryMappedFile>* mmap) {
-  RETURN_NOT_OK(MemoryMappedFile::Create(path, size, mmap));
+                                       std::shared_ptr<MemoryMappedFile>* mmap,
+				       const off_t offset) {
```

Comment:
I think you need to run clang-format
```suggestion
				           const off_t offset) {
```